### PR TITLE
FUSEQE-10939 - Added remaining configs for Slack, Jira, Telegram

### PR DESCRIPTION
**Pull request by lfabriko**
- Split each configuration to separate .feature file (uri, property, secret)
- Jira, Slack, Telegram

### CODE REVIEW

**Review by christophd — COMMENTED** [review]
Looking good! I like the clean separation of features

Some files seem to be obsolete. Please consider to remove those files

File: test/telegram/telegram-source.feature [hunk 1]
```diff
@@ -1,19 +0,0 @@
-Feature: Telegram Kamelet
```
[the remark this answers is not in the document]
Comment (lfabriko) [reply test/telegram/telegram-source.feature:1]: Yes, this is obsolete and will be removed.

File: test/jira/jira-source.feature [hunk 2]
```diff
@@ -1,52 +1,4 @@
...
-    And Camel-K integration jira-to-log-uri-based should print ${summary}
-    And Camel-K integration inmem-to-log should print ${summary}
-
-  Scenario: Remove Camel-K resources
-    Given delete Camel-K integration jira-to-log-secret-based
-    Given delete Camel-K integration jira-to-log-uri-based
-    Given delete Camel-K integration jira-to-inmem
-    Given delete Camel-K integration inmem-to-log
-    Given delete KameletBinding inmem-to-log
+    Given print "TBD"
```
[the remark this answers is not in the document]
Comment (lfabriko) [reply test/jira/jira-source.feature:4]: There I would like to change Camel K integration to KameletBinding (ie. jira-to-inmem.binding.yaml), the integration is used in different test; so I kept the infrastructure and the jira-source is currently just "stub"...

File: test/telegram/telegram-source.feature (hunk 1 above)
[the remark this answers is not in the document]
Comment (lfabriko) [reply test/telegram/telegram-source.feature:1]: This file is already deleted by this pull request...

[the remark this answers is not in the document]
Comment (christophd) [reply test/telegram/telegram-source.feature:1]: of course yes

File: test/jira/jira-source.feature (hunk 2 above)
[the remark this answers is not in the document]
Comment (christophd) [reply test/jira/jira-source.feature:4]: ok fine with it

**Pull request merged**